source/Main.hx

```diff
@@ -3,6 +3,9 @@ package;
 import flixel.FlxGame;
 import flixel.FlxState;
 import funkin.Preferences;
+#if desktop
+import funkin.audio.ALSoftConfig; // Just to make sure DCE doesn't remove this, since it's not directly referenced anywhere else.
```

Comment:
Dead code elimination is disabled in Funkin'
https://github.com/FunkinCrew/Funkin/blob/main/project.hxp#L560-L562